desci-contracts/hardhat.config.ts

```diff
@@ -55,7 +55,7 @@ module.exports = {
       live: false,
       url: "http://127.0.0.1:8545",
       accounts: {
-        mnemonic: "test test test test test test test test test test test junk",
+        mnemonic: process.env.MNEMONIC,
```

Comment:
makes total sense @polus-arcticus -- I would also go ahead and replace the rest of the mnemonics specified in this file with the environment variable